README.md

```diff
@@ -31,10 +30,12 @@ for up to date installation instructions.
 
 [carthage-installation]: https://github.com/Carthage/Carthage#adding-frameworks-to-an-application
 
-You'll also need to add `Runes.framework` to your project. [Runes] is a
-dependency of Argo, so you don't need to specify it in your Cartfile.
+You'll also need to add `Runes.framework` and `Box.framework` to your project.
+[Runes] and [Box] are both Argo dependencies, so you don't need to specify
+them.
```

Comment:
How about something like "so Carthage will grab them for you automatically, you just need to add the resulting frameworks to your project."